app/pb_kits/playbook/pb_avatar/docs/_footer.md

```diff
@@ -0,0 +1,3 @@
+### Things to Avoid
+
+Avoid using Avatar with the user's name right next to it or under it. Use User kit instead.
```

Comment:
You might also say don't use this for images that are not profile pictures...ect